sys/log/full/src/log.c

```diff
@@ -982,6 +1161,42 @@ log_read_hdr(struct log *log, const void *dptr, struct log_entry_hdr *hdr)
     return 0;
 }
 
+int
+log_read_trailer(struct log *log, const void *dptr, uint16_t tlv, void *buf)
```

Comment:
Functions suggest that it can read trailer of certain type designated as `tlv`.
Function on the other hand only works if `tlv == LOG_TLV_NUM_ENTRIES`, it could return error before reading header if tlv is not of the only supported type.